backend/app/modules/uq/core.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Scores:
+    identity_poap: int = 10
+    epoch0_poap: int = 10
+    num_of_donations: int = 10
+    threshold: int = 20
+    min_return: float = 0.2
+    max_return: float = 1.0
+
+
+def calculate_uq(
+    has_epoch_zero_poap: bool,
+    has_identity_poap: bool,
+    num_of_donations: int,
+    gp_score: float,
+    scores: Scores,
+) -> float:
+    uq_score = 0
+
+    if has_identity_poap:
+        uq_score += scores.identity_poap
+    if num_of_donations > 1:
+        uq_score += scores.num_of_donations
```

Comment:
is there a doc where we defined the num of donations is equal to 10 points?

Reply:
https://chat.wildland.dev/wildland/pl/1inmsw3o9br8tqpm47xcizb3xc